css/amp-story-player-shadow.css

```diff
@@ -87,71 +88,89 @@
-  }
-
-  .i-amphtml-story-player-panel-next {
-    margin-inline-end: var(--i-amphtml-story-player-panel-button-margin);
-    background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="16" viewBox="0 0 16 16" fill="white" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.3333 8L0 16V0L11.3333 8ZM16 16H13.3333V0H16V16Z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M11.3333 8L0 16V0L11.3333 8ZM16 16H13.3333V0H16V16Z"/></svg>')!important;
-    right: 0;
-  }
-
-  [dir="rtl"].i-amphtml-story-player-panel-prev {
-    right: 0;
-  }
-
-  [dir="rtl"].i-amphtml-story-player-panel-next {
-    background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="16" viewBox="0 0 16 16" fill="white" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.66667 8L16 16V0L4.66667 8ZM0 16H2.66667V0H0V16Z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M4.66667 8L16 16V0L4.66667 8ZM0 16H2.66667V0H0V16Z"/></svg>')!important;
-    left: 0;
-  }
+.i-amphtml-story-player-panel.i-amphtml-story-player-panel-medium {
+  --i-amphtml-story-player-panel-responsive-margin: 0px;
+  --i-amphtml-story-player-panel-width: calc(
+    var(--i-amphtml-story-player-height) *
+      var(--i-amphtml-story-player-panel-ratio)
+  );
+}
+
+/* Override if iframe height is less than 538px. */
+.i-amphtml-story-player-panel.i-amphtml-story-player-panel-small {
+  /** If changing this, also change amp-story-desktop-one-panel.css **/
+  --i-amphtml-story-player-panel-ratio: 36 / 53;
+}
+
+.i-amphtml-story-player-panel-prev,
+.i-amphtml-story-player-panel-next {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 48px;
+  height: 48px;
+  z-index: 1;
+  background-color: transparent;
+  border: none;
+  background-position: center;
+  background-repeat: no-repeat;
+  cursor: pointer;
+  transition: 150ms opacity;
+}
+
+.i-amphtml-story-player-panel-prev[disabled],
+.i-amphtml-story-player-panel-next[disabled] {
+  opacity: 0.1;
+  cursor: initial;
+}
+
+.i-amphtml-story-player-full-bleed-story .i-amphtml-story-player-panel-prev,
+.i-amphtml-story-player-full-bleed-story .i-amphtml-story-player-panel-next,
+:not(.i-amphtml-story-player-panel) .i-amphtml-story-player-panel-prev,
+:not(.i-amphtml-story-player-panel) .i-amphtml-story-player-panel-next {
+  opacity: 0;
+  pointer-events: none;
+}
+
+.i-amphtml-story-player-panel-prev, .i-amphtml-story-player-panel-next {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" fill="none"><path stroke="#fff" stroke-linecap="round" stroke-width="1.9" d="M17 29.2V18.8c0-1.2 1.4-2 2.5-1.3l7.5 5.2c1 .6 1 2 0 2.6l-7.5 5.2c-1 .7-2.5 0-2.5-1.3Z"/><rect width="1.8" height="15.6" x="30.2" y="16.2" fill="#fff" rx=".9"/></svg>') !important;
```

Comment:
This is the SVG that I effectively changed in this file, the rest of the changes are that I used `translateY(-50%) rotate(180deg)` instead of adding a new SVG for the left arrow.

All the other lines changed were just a formatting issue with the original file where there was extra indentation in most of the file :(